.ci/actions/backporting/action.yml

```diff
@@ -0,0 +1,56 @@
+name: 'Pull Request Backporting'
+description: 'Creates a backporting pull request'
+
+inputs:
+  target-branch:
+    description: "The branch where the cherry-pick must be applied"
+    required: true
+  title:
+    description: "The cherry-pick pull request title"
+    default: "${{ github.event.pull_request.title }}"
+    required: false
+  body:
+    description: "The cherry-pick pull request body"
+    default: "**Backport:** ${{ github.event.pull_request.html_url }}\r\n\r\n${{ github.event.pull_request.body }}"
+    required: false
+  original-pr-branch:
+    description: "Original pull request branch name"
+    default: "${{ github.event.pull_request.head.ref }}"
+    required: false
+  author:
+    description: "Author of the original pull request"
+    default: "${{ github.event.pull_request.user.login }}"
+    required: false
+  additional-reviewers:
```

Comment:
yeah that is, but I'm also having issues getting `github.event.pull_request.requested_reviewers` from the composite action itself (it seems some JSON serialization or something similar), the same issue is not present if it is provided as `input` that;s why I provided it from the workflow [1]. 

Still investigating how to overcome this issue in order to set `github.event.pull_request.requested_reviewers` as default instead of `[]` .

[1] https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2165/files#diff-da3d87773e13508d9ac9396466da2f435481735b38a8f891a22ee42634bf890dR43

Reply:
well, make it dynamic in the action. I mean to keep it as empty by default and in the `Prepare reviewers` step, if the input is empty, get the `github.event.pull_request.requested_reviewers`
and add that behavior into the description of the input
for a first version, that should be enough and can be improved later maybe
wdyt ?

Reply:
yeah that was I tried, but it seems that if I try to get `github.event.pull_request.requested_reviewers` from the composite action it gets `null`, whereas if I provide it from the workflow it works fine (maybe you know why :smile: ). BTW I'll try to find out why it is happening but in case I cannot find it in the short term I think we can keep it as it is: `provide it from the workflow itself`, at least for this first version. wdyt?

Reply:
It seems that right now, as far as I could see and try, is not possible to get github context information from the composite action itself (they resolve to `null`) - therefore the only solution is to keep as default reviewer the `owner` of the original PR and I would add the original PR reviewers in the workflow invocation as already did here https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/2165/files#diff-da3d87773e13508d9ac9396466da2f435481735b38a8f891a22ee42634bf890dR43

Reply:
oh ok. Then let's keep it like that !